File: apps/backend/bookingms/src/main/java/com/example/cargotracker/bookingms/domain/model/events/CargoRoutedEvent.java
package com.example.cargotracker.bookingms.domain.model.events;

import com.example.cargotracker.bookingms.domain.model.valueobjects.CargoItinerary;
import org.axonframework.eventsourcing.annotation.EventTag;

/**
 * 経路が確定し予約に紐付けられたイベント（US11 / UC09）。
 */
public record CargoRoutedEvent(@EventTag String bookingId, CargoItinerary itinerary) {}

File: apps/cargo-tracker/src/CargoTracker.Web/Booking/Domain/Events/CargoRoutedEvent.cs
using CargoTracker.Booking.Domain.Model;
using MediatR;

namespace CargoTracker.Booking.Domain.Events;

/// <summary>確定経路（旅程）が予約に紐付けられたことを表すドメインイベント（US11）。</summary>
public sealed record CargoRoutedEvent(BookingId BookingId) : INotification;
